### PR TITLE
Add DISABLE_LOGGING LOG4XX LOG5XX uwsgi arguments and fix custom log.ini reading

### PR DESCRIPTION
**Pull request by Kuchteq**
Hi, Perhaps we add these 3 new parameters to easily control uwsgi logging capabilities. Also log.ini wasn't being read because later the python part is appended where it is expected that log.ini file is in format log_$HOSTNAME.ini

### CODE REVIEW

File: scripts/env-data.sh [hunk 1]
```diff
@@ -25,8 +25,15 @@ fi
 if [ -z "${ALLOW_LISTING}" ]; then
 	ALLOW_LISTING=True
 fi
-if [ -z "${LOGGING}" ]; then
-	LOGGING=false
+if [ -z "${DISABLE_LOGGING}" ]; then
```
Comment (NyakudyaA) [inline scripts/env-data.sh:28]: What happens https://github.com/kartoza/docker-mapproxy/blob/master/scripts/start.sh#L56 now that logging env is gone

**Review by NyakudyaA — CHANGES_REQUESTED** [review]
Please resolve the comment

File: scripts/env-data.sh (hunk 1 above)
Comment (Kuchteq) [reply scripts/env-data.sh:31]: Good you point it out. I made a typo there, but now it made me wonder how to combine disable logging and regular logging, so i pushed a change where if only logging is specified, disable logging will be the inversion of it, for the envsubst later. There are 4 combinations of logging and disable-logging but one is directly related to uwsgi settings and one to mapproxy's via log.ini so if this was to be introduced there would need to be some clarification in docs.

File: scripts/env-data.sh [hunk 2]
```diff
@@ -28,6 +28,20 @@ fi
 if [ -z "${LOGGING}" ]; then
 	LOGGING=false
 fi
+if [ -z "${DISABLE_LOGGING}" ]; then
+        if [ "${LOGGING}" -eq true ]; then
```
Comment (NyakudyaA) [inline scripts/env-data.sh:32]: Can we do a case-insensitive test like 
```
if [ "${LOGGING}" =~ [Tt][Rr][Uu][Ee] ]; then  
```

**Review by NyakudyaA — APPROVED** [review]

**Comment by Kuchteq** [comment]
Great! Thanks! Btw pipeline is failing cause the python package is supposed to be  python3-dev and not  python-dev. I think it was some recent change with the naming

**Comment by NyakudyaA** [comment]
@Kuchteq I will check it and fix

**Pull request merged**